DataManagementSystem/scripts/dirac-dms-directory-sync.py

```diff
@@ -111,7 +112,7 @@ def getSetOfRemoteSubDirectoriesAndFiles(path,fc,directories,files):
           return S_ERROR('Error: ' + res['Message'])
       return S_OK()
     else:
-      return S_ERROR("Error:" + result['Message'])
+      return S_ERROR("Error: %s" % result['Value'])
```

Comment:
These series of S_ERROR in a recursive method may end up with lots of "Error: " piling up, right?

In case of error, why not just `return result`